universal-login-relayer/lib/http/routes/authorisation.ts

```diff
@@ -3,10 +3,11 @@ import AuthorisationStore, {AuthorisationRequest} from '../../integration/sql/se
 import {asyncHandler, sanitize, responseOf, asString, asObject} from '@restless/restless';
 import {getDeviceInfo} from '../utils/getDeviceInfo';
 import {recoverFromCancelAuthorisationRequest, CancelAuthorisationRequest, hashCancelAuthorisationRequest} from '@universal-login/commons';
-import { ethers, providers} from 'ethers';
+import { ethers } from 'ethers';
```

Comment:
1) Avoid leading & trailing space
2) Why linter let it pass?

Reply:
I guess we should add `"check-module"` to
`"whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-separator",
      "check-type",
      "check-type-operator"
    ]`